### PR TITLE
[SYCLomatic]Enable input style of CommaSeparated for option --extra-arg

### PR DESCRIPTION
**Pull request by wenhuiNi**

### CODE REVIEW

File: clang/test/dpct/extra_arg_comma_separated/header/in_header.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include "cuda_runtime.h"
+
+// CHECK: inline void helloFromGPU2(const cl::sycl::nd_item<3> &item_ct1) {
+// CHECK-NEXT:     int a = item_ct1.get_group(2) * item_ct1.get_local_range(2) + item_ct1.get_local_id(2) + item_ct1.get_group(2) +
+// CHECK-NEXT:     item_ct1.get_local_range(2) + item_ct1.get_local_id(2);
+// CHECK-NEXT: }
+__global__ void helloFromGPU2() {
+  int a = blockIdx.x * blockDim.x + threadIdx.x + blockIdx.x +
+          blockDim.x + threadIdx.x;
+}
```
Comment (yihwang-nv) [inline clang/test/dpct/extra_arg_comma_separated/header/in_header.h]: Please add new line

File: clang/test/dpct/extra_arg_comma_separated/header2/out_header.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#include "cuda_runtime.h"
+
+// CHECK: inline void helloFromGPU(int i, const cl::sycl::nd_item<3> &item_ct1) {
+// CHECK-NEXT:     int a = item_ct1.get_group(2) * item_ct1.get_local_range(2) + item_ct1.get_local_id(2) + item_ct1.get_group(2) +
+// CHECK-NEXT:     item_ct1.get_local_range(2) + item_ct1.get_local_id(2);
+// CHECK-NEXT: }
+__global__ void helloFromGPU() {
+  int a = blockIdx.x * blockDim.x + threadIdx.x + blockIdx.x +
+          blockDim.x + threadIdx.x;
+}
```
Comment (yihwang-nv) [inline clang/test/dpct/extra_arg_comma_separated/header2/out_header.h]: Please add new line

File: clang/test/dpct/extra_arg_comma_separated/source_file/test.cu [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  helloFromGPU <<<1, 1>>>();
+  // CHECK: q_ct1.parallel_for(
+  // CHECK-NEXT: sycl::nd_range<3>(sycl::range<3>(1, 1, 1), sycl::range<3>(1, 1, 1)), 
+  // CHECK-NEXT: [=](sycl::nd_item<3> item_ct1) {
+  // CHECK-NEXT:   helloFromGPU2();
+  // CHECK-NEXT: });
+  helloFromGPU2 <<<1, 1>>>();
+
+  return 0;
+}
```
Comment (yihwang-nv) [inline clang/test/dpct/extra_arg_comma_separated/source_file/test.cu]: Please add new line

**Review by yihwang-nv — APPROVED** [review]

**Review by ziranzha — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**